### PR TITLE
Operator help code optimisation

### PR DESCRIPTION
**Pull request by khalidmammadov**
This PR an improvement to the codebase.
It removes duplicate references to the dictionary object and removes the noise and reduces the code
 
---
**^ Add meaningful description above**

Read the **[Pull Request Guidelines](https://github.com/apache/airflow/blob/main/CONTRIBUTING.rst#pull-request-guidelines)** for more information.
In case of fundamental code change, Airflow Improvement Proposal ([AIP](https://cwiki.apache.org/confluence/display/AIRFLOW/Airflow+Improvements+Proposals)) is needed.
In case of a new dependency, check compliance with the [ASF 3rd Party License Policy](https://www.apache.org/legal/resolved.html#category-x).
In case of backwards incompatible changes please leave a note in [UPDATING.md](https://github.com/apache/airflow/blob/main/UPDATING.md).

### CODE REVIEW

**Comment by khalidmammadov** [comment]
@ephraimbuddy Can you please take a look?

File: airflow/utils/operator_helpers.py [hunk 1]
```diff
@@ -58,37 +59,32 @@ def context_to_airflow_vars(context, in_env_var_format=False):
...
+        (dag_run, 'run_id', 'AIRFLOW_CONTEXT_DAG_RUN_ID'),
+    ]
+
+    for subject, attr, mapping_key in ops:
+        _attr = getattr(subject, attr, None)
+        if subject and _attr:
+            mapping_value = AIRFLOW_VAR_NAME_FORMAT_MAPPING[mapping_key][name_format]
+            if isinstance(_attr, str):
+                params[mapping_value] = _attr
+            if isinstance(_attr, datetime):
```
Comment (potiuk) [inline airflow/utils/operator_helpers.py]:
```suggestion
            elif isinstance(_attr, datetime):
```

**Review by potiuk — CHANGES_REQUESTED** [review]
looks cool! one small nit

File: airflow/utils/operator_helpers.py (hunk 1 above)
Comment (khalidmammadov) [reply airflow/utils/operator_helpers.py]: Thanks for spotting! Fixed.

**Review by potiuk — APPROVED** [review]

**Comment by Zam83-AZE** [comment]
Tebrikler Juji)

**Pull request merged**